factory/glideFactoryCredentials.py

```diff
@@ -268,7 +268,7 @@ def validate_frontend(classad, frontend_descript, pub_key_obj):
     return sym_key_obj, frontend_sec_name
 
 
-def check_security_credentials(auth_method, params, client_int_name, entry_name):
+def check_security_credentials(auth_method, params, client_int_name, entry_name, scitoken_passthru=False):
     """
     Verify taht only credentials for the given auth method are in the params
 
```

Comment:
Add also `scitoken_passthru` to the parameters and possibly convert to the google format.
```
scitoken_passthru (bool): skip the GlideinProxy requirement and allow to use only scitokens also for grid_proxy authentication method
```

Reply:
done